scripts/bower/publish.sh

```diff
@@ -75,6 +75,29 @@ function prepare {
     replaceJsonProp "bower.json" "angular.*" ".*" "$NEW_VERSION"
     replaceJsonProp "package.json" "version" ".*" "$NEW_VERSION"
     replaceJsonProp "package.json" "angular.*" ".*" "$NEW_VERSION"
+    deleteJsonProp "package.json" "main"
+
+    echo "-- Adding CommonJS index file"
+    if [ -f "index.js" ]
+    then
+      rm index.js
+    fi
+
+    touch index.js
+    echo "require('./$repo');" >> index.js
```

Comment:
instead of removing index.js, touching it and then appending to it, you could just do this:

`echo "require('./$repo');" > index.js` --- that would shave off a few lines of code and be a bit easier to read